Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/project/.gitignore
rename from .gitignore
rename to project/.gitignore
--- a/.gitignore	(revision 3487d4ca21fa82af3ad86e4387422fe7d257bf68)
+++ b/project/.gitignore	(date 1655825133341)
@@ -1,4 +1,4 @@
-.idea/
+../.idea/
 ./idea/*
 ./database/database.db
 *.xml
\ No newline at end of file
